api/search.go

```diff
@@ -57,6 +58,13 @@ func (api *SearchAPI) getSearch(w http.ResponseWriter, r *http.Request) {
 
 	log.Info("incoming request", logData)
 
+	auditParams := common.Params{"dataset_id": datasetID, "edition": edition, "version": version, "dimension": dimension}
+
+	if err := api.auditor.Record(r.Context(), "getSearch", "attempted", auditParams); err != nil {
```

Comment:
`getSearch` and `attempted` would be better defined as constants at the top of the file